my-app/src/components/login/login.js

```diff
@@ -41,6 +41,28 @@ class Login extends React.Component {
     .catch(err => console.error(err));
   }
 
+  getUserServer = _ => {
+    const userName = this.state.username;
+    const password = this.state.password;
+    fetch("http://localhost:5000/users/", {
+      method: 'get',
+      headers: {'Content-Type': 'application/json'},
+      body: JSON.stringify({
+        "userName": userName,
+        "password": password
+      })
+    })
+    .then((response) => {
+      if(response.status === 404) {
+        alert('Failed to authenticate user');
+      }
```

Comment:
There should be some response on the frontend if a user is not authenticated. This function just allows the user to sign in regardless and sends a backend signal.

Reply:
For now it just gives you an alert but the goal was to make a simple connection between the backend and the frontend and not handle the actual response, I do agree though that is something that we need to do.

Reply:
Fixed the issue, if user does get authenticated he/she gets redirected to homepage, otherwise they get an alert.